google-datacatalog-rdbms-connector/src/google/datacatalog_connectors/rdbms/prepare/sql_objects/datacatalog_sql_objects_entry_factory.py

```diff
@@ -0,0 +1,164 @@
+
+        self.__set_entry_system_timestamps(entry, sql_object_fields,
+                                           sql_object_item)
+
+        self.__set_entry_description(entry, sql_object_fields, sql_object_item)
+
+        entry.name = datacatalog.DataCatalogClient.entry_path(
+            self.__project_id, self.__location_id, self.__entry_group_id,
+            entry_id)
+
+        entry.linked_resource = '//{}//{}'.format(self.__metadata_host_server,
+                                                  entry_id)
+
+        return entry_id, entry
+
+    @classmethod
+    def __filter_entry_model_fields(cls, sql_object_fields):
+        sql_object_fields = [
+            field for field in sql_object_fields
+            if field[config_constants.SQL_OBJECT_FIELD_TARGET][
+                config_constants.SQL_OBJECT_FIELD_TARGET_MODEL] ==
+            config_constants.SQL_OBJECT_ENTRY_MODEL
+        ]
+        return sql_object_fields
+
+    @classmethod
+    def __set_entry_system_timestamps(cls, entry, sql_object_fields,
+                                      sql_object_item):
+
+        created_time_field = cls.__find_sql_object_field(
+            sql_object_fields, config_constants.SQL_OBJECT_ENTRY_CREATE_TIME)
+
+        if created_time_field:
+            created_time = cls.__get_sql_object_field_value(
+                sql_object_item, created_time_field)
+
+            update_time_field = cls.__find_sql_object_field(
+                sql_object_fields,
+                config_constants.SQL_OBJECT_ENTRY_UPDATE_TIME)
+
+            update_time = None
+            if update_time_field:
+                update_time = cls.__get_sql_object_field_value(
+                    sql_object_item, update_time_field)
+
+            create_time, update_time = \
+                cls.__convert_source_system_timestamp_fields(
+                    created_time,
+                    update_time)
+
+            if create_time and update_time:
+                created_timestamp = timestamp_pb2.Timestamp()
+                created_timestamp.FromSeconds(create_time)
+                entry.source_system_timestamps.create_time = created_timestamp
+
+                updated_timestamp = timestamp_pb2.Timestamp()
+                updated_timestamp.FromSeconds(update_time)
+                entry.source_system_timestamps.update_time = updated_timestamp
+
+    @classmethod
+    def __set_entry_description(cls, entry, sql_object_fields,
+                                sql_object_item):
+        description_field = cls.__find_sql_object_field(
+            sql_object_fields, config_constants.SQL_OBJECT_ENTRY_DESCRIPTION)
+
+        if description_field:
+            description = sql_object_item.get(
+                description_field[config_constants.SQL_OBJECT_FIELD_TARGET][
+                    config_constants.SQL_OBJECT_FIELD_TARGET_NAME])
+
+            if pd.isna(description):
+                description = ''
+
+            entry.description = description
```

Comment:
```suggestion
            entry.description = description if not pd.isna(description) else ''
```
WDYT about using a one-liner here?

Reply:
I think in this particular case, using a simple if is easier to understand the code flow, and the one-liner has 80 chars, so I would need to split the line. If it's ok with you, I would like to keep it the way it is.